### ISSUE TITLE
Allow font size change or more compact view

### ISSUE THREAD

**Issue by gerv** [google/google-authenticator-android#33]
I have more than one screen of passwords; I'd love it to only be a screenful. The best layout would be:

Google: foo@gmail.com 123456

with the expiry time as a thin "progress bar" at the top of the screen, given that it's the same for all the codes (at least for me). (If there's some way of it being different, then it could be a thin bar under each line.)

With a layout like that, you could fit 10 or 12 codes on the screen and still have them as readable. Although also, assigning each a random pastel colour would also help to distinguish them, and making the grey text a lighter grey would improve matters too.

Gerv

**Comment by ocdtrekkie** [comment]
As an important note: This is not the Play Store version, and the Play Store version will not get anything from this repo. (FYI: The Play Store version is closed source/proprietary now. The last open source version was 2.21.)

That being said, I personally would love some display options for this.

**Comment by gerv** [comment]
Yes, I know. (I use the version from F-Droid, which I believe is compiled from this repo.)

**Comment by ocdtrekkie** [comment]
Ah, okay, yes the F-Droid version should be from here. :D 

I would totally love to see this code too.

**Comment by ThomasHabets** [comment]
Optional screen layout change pull requests would be accepted. :-)

**Comment by gerv** [comment]
Cool; although I've never worked on Android code in my life, so people should not hold their breath expecting this to come from me... :-((
